File: easy/testing.rb
m = ["Done", "Low", "Medium", "High", "Critical"]

def prio(a)
  case a
  when 0
    0
  when 1..2
    1
  when 3..4
    2
  when 5..6
    3
  else
    4
  end
end

File.open(ARGV[0]).each_line do |line|
  s = line.chomp.split(" | ")
  r = 0
  for ix in (0...s[0].length)
    r += 1 if s[0][ix] != s[1][ix]
  end
  puts m[prio(r)]
end

File: easy/testing.go
package main

import (
	"bufio"
	"fmt"
	"log"
	"os"
	"strings"
)

func main() {
	var t string
	data, err := os.Open(os.Args[1])
	if err != nil {
		log.Fatal(err)
	}
	defer data.Close()
	scanner := bufio.NewScanner(data)
	for scanner.Scan() {
		var r int
		s := strings.Split(scanner.Text(), " | ")
		for i := 0; i < len(s[0]); i++ {
			if s[0][i] != s[1][i] {
				r++
				if r > 6 {
					break
				}
			}
		}
		switch {
		case r == 0:
			t = "Done"
		case r <= 2:
			t = "Low"
		case r <= 4:
			t = "Medium"
		case r <= 6:
			t = "High"
		default:
			t = "Critical"
		}
		fmt.Println(t)
	}
}
